wasm2map/src/lib.rs

```diff
@@ -155,9 +155,17 @@ impl WASM {
                         // The directory index 0 is defined to correspond to the compilation unit directory.
                         if file.directory_index() != 0 {
```

Comment:
I think we should remove this if check. WDYT?

Reply:
I agree, this can be removed.

Reply:
Done.